content/api/loaders.md

```diff
@@ -3,4 +3,302 @@ title: Loader API
 sort: 4
 ---
 
-> TODO
+Loaders allow you to preprocess files as you require() or “load” them. Loaders are kind of like “tasks” in other build tools, 
+and provide a powerful way to handle frontend build steps. Loaders can transform files from a different language like, CoffeeScript to JavaScript, or inline images as data URLs. Loaders even allow you to do things like require() css files right in your JavaScript!
+
+To tell webpack to transform a module with a loader, you can specify the loader in the webpack [configuration](../configuration) file (preferred) or in the module request, such as in a `require()` call.
+
+## How to write a loader
+
+A loader is just a file that exports a function. The compiler calls this function and passes the result of the previous loader or the resource file into it. The this context of the function is filled-in by the compiler with some useful methods that allow the loader to, among other things, change its invocation style to async or get query parameters. The first loader is passed one argument: the content of the resource file. The compiler expects a result from the last loader. The result should be a String or a Buffer (which is converted to a string), representing the JavaScript source code of the module. An optional SourceMap result (as JSON object) may also be passed.
```

Comment:
s/The this/The `this`/
(I didn't know at first if you were talking about the JS `this` or real-life this 😄)